po/ar.po

```diff
@@ -6594,10 +6594,10 @@ msgstr ""
 msgid "Djibouti"
 msgstr "جيبوتي"
 
-msgid "Do center DVB subs on this service"
+msgid "Center DVB Subs On This Service"
 msgstr ""
 
-msgid "Do not center DVB subs on this service"
+msgid "Don't Center DVB Subs On This Service"
```

Comment:
(This and all others):
Menu items shouldn't be capitalised, only headings and brand names - eg.
`Don't Center DVB Subs On This Service` -> `Don't center DVB subs on this service`

English doesn't follow the German-style capitalisation of nouns.

Reply:
Menu entries are being capitalised.  (Titles, menus and button labels are being displayed capitalised. Descriptions, help text, etc are normal sentence style.)